### ISSUE TITLE
[SFS] Save this factsheet button in Firefox does nothing. 

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oekg#21]
## Description of the issue
Tried to add a factsheet using Firefox on Windows 10. The _save factsheet_ button does absolutely nothing.
![grafik](https://user-images.githubusercontent.com/36844029/228225853-27a1867e-d2c0-40ff-9311-fdd61921e32e.png)

(working in Microsoft Edge now instead to test functionality)

## Steps to Reproduce
1. got to: http://toep.iks.cs.ovgu.de/factsheet/
2. Click _add a new_
3. Enter factsheet acronym
4. press save this factsheet button

## Ideas of solution

Describe possible ideas for solution and evaluate advantages and disadvantages.

## Context and Environment
* Version used:  toep
* Operating system: Windows 10
* Environment setup and (python) version: 

## Workflow checklist
- [ ] I am aware of the workflow in [CONTRIBUTING.md](CONTRIBUTING.md)

**Comment by christian-rli** [comment]
I cannot reproduce your error in firefox on linux. The button is without function in the overview mode. In edit mode, a green message at the bottom of the screen appears, informing me of a successful change. Unless there is a specific error for windows, I think hiding the button in the overview mode should be the solution to this issue. Feel free to correct me, though.

**Comment by han-f** [comment]
I tried it again. I added information, a blue message on the bottom indicated this would be saved when saving. I clicked the green save button and nothing happened. Well, at least no success message appeared after clicking the button.
When I used the back arrow on the top left to get back to the overview, a message in green said it saved the sheet. I could not reproduce this a second time. The second time it took a minute or so after clicking the save button for the message to appear. So it seems to work in Firefox Windows, albeit with a delay.

But: it did not save my change. I deleted contact persons and then added only a single one (it was two before). After saving both appear again in the overview, also after refreshing the browser.

**Comment by christian-rli** [comment]
Hm. Have you tried to clear your cache when reloading? The easiest way of doing that is to hold down the shift-key while clicking the reload symbol.

**Comment by christian-rli** [comment]
... I can now reproduce this behavior. It seemed to work for my first edits, but I noticed that after hitting the save button and selecting overview, the view appears with the selected changes. However, when refreshing the site shows the previous, old (initial?) state.

**Comment by han-f** [comment]
I also revisited the page, emptied cache and reloaded and still the old entries show, they also persist in the edit mode, even after having had them deleted. I also reproduce this behaviour in Windows and Edge browser.

It seems deletions do not make it through.

**Comment by adelmemariani** [comment]
Thanks for reposting the bug about the contact person. I fixed it. 

I tried the factsheets with Firefox, and they worked. Removing the save button from overview mode is a good idea. I will include this update in the next release. 

But, when a factsheet is too large, it may take some time to see the green message about the saved factsheet. Basically, the save button sends your factsheet to the backend, and the backend processes it a bit and stores it in Apache Jena as triples. Only after completion of these processes a message indicating a successful save/update or an error will appear on the bottom-left-side of the page.
